src/tests/test_examples.rs

```diff
@@ -51,7 +51,7 @@ fn test_directory(dir_in: &str) -> Result<(), Error> {
             .spawn()?
             .wait()?
             .success();
-        assert_eq!(success, true, "{:?}", &in_file);
```

Comment:
This is out of scope for this patch

Reply:
Superseded by 47ccd2fcb2f2f51ad6f95e3fdc11a2ddbdece8fa